sherpa/sim/sample.py

```diff
@@ -199,45 +220,61 @@ def get_scales(self, fit, myscales=None):
                     fit.estmethod = conf
                     try:
                         t = fit.est_errors(parlist=(par,))
-                        if t.parmins[0] is not None and t.parmaxes[0] is not None:
-                            scale = numpy.abs(t.parmins[0])
-
-                        else:
 
-                            if t.parmins[0] is None and t.parmaxes[0] is not None:
-                                scale = numpy.abs(t.parmaxes[0])
+                        # QUS: could this not be
```

Comment:
This code seems to just look at the first value or parmin/parmax and say, "hey, let's get a reasonable guess. Let's try parmin. Uups, that's a None. Well, we'll take parmax instead."
I;ve done very similar things in hand-written code for analysis. It's kind of hand-wavy. I don't think we can do much better though (could at least take the max or mean of [parmax,parmin] is both are available), but fundamentally, I'm not sure can can do much better without actually tracking a distribution of numbers.

So, would that belong into this PR or not?

Reply:
I created

- #2340

to track this, so I don't think we want to change it here.